units/ftdc_rollups.go

```diff
@@ -0,0 +1,129 @@
+package units
+
+import (
+	"context"
+
+	"github.com/evergreen-ci/cedar"
+	"github.com/evergreen-ci/cedar/model"
+	"github.com/evergreen-ci/cedar/perf"
+	"github.com/mongodb/amboy"
+	"github.com/mongodb/amboy/dependency"
+	"github.com/mongodb/amboy/job"
+	"github.com/mongodb/amboy/registry"
+	"github.com/mongodb/ftdc"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+)
+
+const (
+	ftdcRollupsJobName = "ftdc-rollups"
+)
+
+type ftdcRollups struct {
+	PerfID       string              `bson:"perf_id" json:"perf_id" yaml:"perf_id"`
+	ArtifactInfo *model.ArtifactInfo `bson:"artifact" json:"artifact" yaml:"artifact"`
+
+	*job.Base `bson:"metadata" json:"metadata" yaml:"metadata"`
+}
+
+func init() {
+	registry.AddJobType(ftdcRollupsJobName, func() amboy.Job {
+		fr := &ftdcRollups{}
+		fr.setup()
+		fr.SetDependency(dependency.NewAlways())
+
+		return fr
+	})
```

Comment:
I'd have a seperate private and follow the pattern that the evergreen versions have/use.